.changeset/config.json

```diff
@@ -6,5 +6,5 @@
   "access": "restricted",
   "baseBranch": "master",
   "updateInternalDependencies": "patch",
-  "ignore": ["@graphql-codegen/testing", "@graphql-codegen/website", "example-*"]
```

Comment:
I renamed `@graphql-codegen/website` to `website` to make consistency between websites, and for `yarn workspace website build`, proposition by @n1ru4l